packages/server/lib/controllers/connection.controller.ts

```diff
@@ -30,7 +32,7 @@ class ConnectionController {
 
             const integration_key = process.env['NANGO_SLACK_INTEGRATION_KEY'] || 'slack';
             const nangoAdminUUID = NANGO_ADMIN_UUID;
-            const env = 'prod';
+            const env = process.env['NANGO_ADMIN_PROD_ENV'] || 'prod';
```

Comment:
we now prefer to use the validated and type safed `parsedEnvs`. Example: https://github.com/NangoHQ/nango/blob/5ea7299ab539532e1f5195b0f98d94cf96e3b618/packages/server/lib/server.ts#L33

Reply:
yes, and that way you can put a default value too